bionic/protocols.py

```diff
@@ -15,12 +15,15 @@
 import sys
 import tempfile
 import shutil
+import warnings
 
+import dask.dataframe as dd
```

Comment:
Can we make this work like `ImageProtocol`, where (a) we do an optional import of `dask.dataframe`, so things still work even when Dask is not installed, and (b) include `DaskProtocol` in `flow.DEFAULT_PROTOCOL` so that the user doesn't have to explicitly specify it?

Reply:
Sg, thanks for the help!